### ISSUE TITLE
Donations

### ISSUE THREAD

**Issue by Laserlicht** [vcmi/vcmi#3795]
I've already seen a few people asking about it.

**Comment by DjWarmonger** [comment]
I believe it's a good moment to start this discussion.

- One obvious example is hosting. Please remind me how does this Digital Ocean work at the moment?
- Just yesterday there was discussion about secure servers hosted for games, or only for a tournament(s).
- Also I still have this AI chatbot with pay-as-you-go API, which kind restricts its use. And that's the common theme of all AI-assisted tools, they are coming but not for free.

**Comment by IvanSavenko** [comment]
> (Patreon, Paypal, ...)

I would also consider services like buy me a coffee or ko-fi, that support one-time donations and not just regular membership like Patreon requires. 

> Please remind me how does this Digital Ocean work at the moment?

What do you mean by 'how'? If you are asking about the cost, then from what I understand our servers cost around 25 USD / month, paid by SXX. Could be optimized to be honest, if somebody is willing to do necessary server management work. DigitalOcean has free 'credits' for open-source projects, but no idea whether we are enrolled into it or how much it covers. So actual costs might be way lower.

**Comment by IvanSavenko** [comment]
So what does everyone in the team thinks about donations?
- is there anyone who feels that we should not accept them?
- what should we use donations for? Server costs are obvious pick, AppStore, dedicated game servers or that AI bot are potential picks, but probably secondary.
- where / in which service should we accept donations? Ideally we should allow both regular and one-time donations, preferably on one service.
- who is going to manage them? Can look into it myself, but my options might be limited - not every service operates in Ukraine.

**Comment by kambala-decapitator** [comment]
donations would be good, can be spent on servers as primary cost, AppStore would also be good.

I think it's better to set them up in EU / USA.

also I think it would be great to try to create a non-profit organization, but again that's basically limited to EU / USA. For example, non-profits don't pay for AppStore.

**Comment by dydzio0614** [comment]
I am OK with donations, extra funds would go for securing the future of paid services

If donations get unexpectedly successful we might think about some new ways to spend money (promotion? giving tips to contributors that can be contacted based on amount of their contribution, starting from part contributors?)

**Comment by dydzio0614** [comment]
also if some contributor want to setup additional private donations i personally would not oppose, as long as link will be properly described and there will be no bad faith (such as trying to farm more money)

**Comment by Laserlicht** [comment]
> * Also I still have this AI chatbot with pay-as-you-go API, which kind restricts its use. And that's the common theme of all AI-assisted tools, they are coming but not for free.

Yes, https://elevenlabs.io/ comes in my mind here. To generate spoken voices for Campaign intros in languages where no official voice exists.

> I would also consider services like buy me a coffee or ko-fi, that support one-time donations and not just regular membership like Patreon requires.

Good point.

> is there anyone who feels that we should not accept them?

Okay from my side.

**Comment by IvanSavenko** [comment]
> I think it's better to set them up in EU / USA.

If somebody from EU / USA is willing to set them up - sure.

> also I think it would be great to try to create a non-profit organization

AFAIK settings up non-profit is a pain. So not worth it for small teams like ours.

> If donations get unexpectedly successful we might think about some new ways to spend money (promotion? giving tips to contributors that can be contacted based on amount of their contribution, starting from part contributors?)

Perhaps, but this is too long-term plan to consider it now. We'll need to cover existing costs, other desired stuff (iOS / game servers / etc), create some buffer so we can cover at least few months ahead in case if donations go down. And all of this is already big "if".

**Comment by ryfactor** [comment]
I for one am happy to donate to get VCMI into App Store.

**Comment by Sam-Kruglov** [comment]
I've seen other projects on github use donations on a specific issue to "sponsor" it. Maybe if this gets set up, people will kind of put bounties on certain tasks which will attract devs. Best I could google https://docs.github.com/en/sponsors/getting-started-with-github-sponsors/about-github-sponsors

Idk anything about that, just a thought

**Comment by iyerusad** [comment]
Would [OpenCollective](https://opencollective.com/) make sense? It seems to provide some structure around collaborative projects without going hard into setting up formal organizations. Not sure how it compares to Github Sponsors. 

Looking over the [JellyFin OpenCollective page](https://opencollective.com/jellyfin), see the following:
- Ability to take in contributions
- Ability to distribute/pay for project expenses
- Ability to accumulate (handle feast/famine cycles)
- Easy Transparency (this is particularly important because most people tend to underestimate costs and overestimate incoming cashflow)
  - Story around [OpenSSL donations](http://veridicalsystems.com/blog/of-money-responsibility-and-pride/) interesting read, arguably happened because most people didn't know/didn't have easy insight.
- UI for all of the above -- this is worth something/don't have to self build

Might be one of the few ways to have packaged options continue to survive if something happens to current project host (or the hosting country).

Its possible the long term reality for this kind of project will always be starving for basic funding -- even paid mobile apps anecdotally report revenues drop off to the point of no longer worth maintaining overhead and project is killed off/Apple Dev license expires. If donations and revenues are allowed to accumulate into something like OpenCollective, that transparency may give project reasonable chance to continue getting basic costs covered for longer.

**Comment by jaicmatija** [comment]
Maybe do something like `fheroes2` guys did, just [organize patreon](https://www.patreon.com/c/fheroes2/home?utm_source=campaign_link&utm_medium=email&utm_campaign=free_member_digest&utm_id=bd49aca6-e4a9-4f0a-9377-83b3005067fd)? Keep it simple, even though I get the point of one-time donations so maybe have 2 models?

**Comment by misiokles** [comment]
What's current status of Donate feature?
